akka-docs/src/main/paradox/persistence-query-durable-state.md

```diff
@@ -35,7 +35,7 @@ durable state is stored in a database and fetched as an asynchronous stream to t
 durable state, provided by the `DurableStateStoreQuery` interface, is used to implement tag based searches in 
 Akka Projections. 
 
-The example below shows how to get the  `DurableStateStoreQuery` from the DurableStateStoreRegistry extension.
+The example below shows how to get the  `DurableStateStoreQuery` from the `DurableStateStoreRegistry` extension.
```

Comment:
For future reference: 
```suggestion
The example below shows how to get the  `DurableStateStoreQuery` from the @apidoc[DurableStateStoreRegistry$] extension.
```
is even nicer as it creates a link to the scaladoc. This is already :+1:  though :)